augmentedreality/Common/include/Android/ArCoreWrapper.h

```diff
@@ -42,8 +42,10 @@ class ArCoreFrameRenderer;
 class ArCorePointCloudRenderer;
 class ArCorePlaneRenderer;
 
-class ArCoreWrapper
+class ArCoreWrapper : public QObject
```

Comment:
Why did you add inheritance from QObject?

Reply:
Ok, maybe it's for `requestPermission`?

In this case, as `this` is not used and context is used only for object lifetime, maybe you can use `&loop` as context and don't derive from `QObject`

Reply:
Ah I thought I needed that for the call into qApp but it looks like I dont